integration_test/src/main.rs

```diff
@@ -1422,6 +1424,38 @@ fn test_get_index_info(cl: &Client) {
     }
 }
 
+fn test_get_zmq_notifications(cl: &Client) {
+    let mut zmq_info = cl.get_zmq_notifications().unwrap();
+
+    // it doesn't matter in which order Bitcoin Core returns the result,
+    // but checking it is easier if it has a known order
+    zmq_info.sort_by(|a, b| {
```

Comment:
In d4983f58cbe1cc2b457890c21a68437a65d8d891:

Why not use `sort_by_key`?

Reply:
good point, resolved

Reply:
oh wait now i remember, `sort_by_key` does not allow returning references to parameters

Reply:
i added a comment in the code explaining this